Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_037.SqlCredentials.sarif

```diff
@@ -205,11 +205,11 @@
             }
           ],
           "fingerprints": {
-            "AssetFingerprint/v1": "[host=servername2.database.windows.net,1433][id=username2][part=servers][platform=Azure][resource=database2]",
-            "ValidationFingerprint/v1": "[host=servername2.database.windows.net,1433][id=username2][resource=database2][secret=password2]",
-            "ValidationFingerprintHash/v1": "d79c54c338cf6b8786b93457bee7bed8e7090750f873a0ce06ed4dca3431acd0",
-            "AssetFingerprint/v2": "{\"host\":\"servername2.database.windows.net,1433\",\"id\":\"username2\",\"part\":\"servers\",\"platform\":\"Azure\",\"resource\":\"database2\"}",
-            "ValidationFingerprint/v2": "{\"host\":\"servername2.database.windows.net,1433\",\"id\":\"username2\",\"resource\":\"database2\",\"secret\":\"password2\"}"
+            "AssetFingerprint/v1": "[host=servername2.database.windows.net][id=username2][part=servers][platform=Azure][port=1433][resource=database2]",
+            "ValidationFingerprint/v1": "[host=servername2.database.windows.net][id=username2][port=1433][resource=database2][secret=password2]",
+            "ValidationFingerprintHash/v1": "d8f735ee725f3c05e20b44ba9a8004f07330d1feb965d2ec56ee35f82edb22f6",
+            "AssetFingerprint/v2": "{\"host\":\"servername2.database.windows.net\",\"id\":\"username2\",\"part\":\"servers\",\"platform\":\"Azure\",\"port\":\"1433\",\"resource\":\"database2\"}",
```

Comment:
This is wrong, the asset fingerprint should not contain the port. The same asset accessed via two distinct ports, is a single asset.

Reply:
Interesting! We weren't removing port from the AssetFingerprint.
Just made the change to enable this.